Cargo.toml

```diff
@@ -16,6 +16,7 @@ default = ["std", "zeroize"]
 std = ["getrandom/std", "base64/std"]
 alloc = ["base64/alloc", "getrandom"]
 js = ["getrandom/js"]
+error_on_truncation = []
```

Comment:
This feature should be documented in the README

Reply:
README and errors.rs updated to indicate that Truncation error is used only when `error_on_truncation` is enabled.